### ISSUE TITLE
Fix documentation build process

### ISSUE THREAD

**Issue by barankyle** [XRFoundation/XREngine#3402]
Documentation stopped building properly starting with #3236, where typedoc was updated from 0.21.x to 0.22.x. I've tried to just change the versions of typedoc, typedoc-plugin-markdown, and docusaurus-plugin-typedoc back to what they were before that commit, but there are still doc build errors.

When I run the build command locally from a clean repo and those three packages rolled back, I get this error:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:363:5)
    at validateString (node:internal/validators:119:11)
    at Object.dirname (node:path:1276:5)
    at MarkedPlugin.getRelativeUrl (/home/kyle/Development/Forks/xrengine/node_modules/typedoc/dist/lib/output/components.js:48:49)
    at DeclarationReflection.<anonymous> (/home/kyle/Development/Forks/xrengine/node_modules/typedoc/dist/lib/output/plugins/MarkedPlugin.js:83:70)
    at DeclarationReflection.wrapper (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
    at eval (eval at createFunctionContext (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:262:23), <anonymous>:13:112)
    at prog (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
    at execIteration (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
    at ReflectionGroup.<anonymous> (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:61:13)
    at ReflectionGroup.wrapper (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
    at eval (eval at createFunctionContext (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:262:23), <anonymous>:13:49)
    at Object.prog [as inverse] (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
    at ReflectionGroup.<anonymous> (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/helpers/if.js:27:22)
    at ReflectionGroup.wrapper (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
    at eval (eval at createFunctionContext (/home/kyle/Development/Forks/xrengine/node_modules/handlebars/dist/cjs/handlebars/compiler/javascript-compiler.js:262:23), <anonymous>:13:47)
```

It looks like the relative file modules/index.html is referenced by the main.js built into docs/docs/docs-common/assets/js/; I'm not sure where this file is supposed to be. Oddly, if I run `npm run build` again, it never tries to reference this file, and the build process finishes without error.

However, I only ever see a /docs-common directory built, and no docs-<package> for any other package. Not sure if this is correct, but I imagine either they all should have a docs- directory, or none should. There's also many other errors and warnings that don't end the build process, but I'm not sure if any of them are all that relevant.

When I try to build the docs using the latest version of typedoc, it doesn't build the sidebar files that sidebars.js is looking for, nor anything in the .docusaurus directory. Not sure if the latest version is unusable or just something else needs to be tweaked.

**Comment by hanzlamateen** [comment]
@barankyle this is is fixed now. You can close it now

**Comment by barankyle** [comment]
Resolved by #4220
